.github/pull_request_template.md

```diff
@@ -17,12 +17,7 @@ If manual verifications required, please provide an environment where the review
 
 ## Checklist (Definition of Done)
 <!-- Please strikethrough options not relevant using two tildes ~~Text~~. Do not delete non relevant options -->
-- [ ] All acceptance criteria specified in JIRA have been completed
-- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
```

Comment:
Do we want to keep the test requirement and only remove the OCM part?

Reply:
Yes, we could keep unit tests.

Integration tests will continue to fail for various reasons .eg secrets and the main one is for most of the endpoints custom  logic will need to be introduce. Thus why I want to relax the requirement of having the test passing. Of course, the team starting a new service will need to make sure that all the tests are passing and are part of their CI&CD pipelines.